tendermint/src/rpc/event_listener.rs

```diff
@@ -101,7 +101,8 @@ impl EventListener {
             dbg!(&msg.to_string());
             return Ok(None);
         }
-
+        dbg!("received neither event nor generic string message:");
+        dbg!(&msg.to_string());
```

Comment:
I checked this with IBC relayer and it's ""

Reply:
I think that is a ping message:
https://github.com/tendermint/tendermint/blob/ba3a2dde376f708085726cd778bb361bd8c7c692/rpc/jsonrpc/server/ws_handler.go#L407

 If the server doesn't drop your event subscription, you can ignore it for now.

Reply:
> If the server doesn't drop your event subscription, you can ignore it for now.

Yes, I am ignoring but was wondering if this could be handled more gracefully :) Anyway..this is for the other issue. Just saw you added this here and wanted to mention that for the failure I see it is empty string.

Reply:
> Yes, I am ignoring but was wondering if this could be handled more gracefully :)

Absolutely, the client should actually play ping-pong with the server instead of treating this as an error.